zlp-scheduler/app/controllers/users_controller.rb

```diff
@@ -46,6 +46,18 @@ def delete_user
     redirect_to manage_cohorts_path()
   end
   
+  def create_temp_student(row)
+    user = User.new
+    user.firstname = row[0]
+    user.lastname = row[1]
+    user.uin = row[2]
+    user.email = row[3]
+    user.role = 'student'
+    user.password = 'Temp'
+    user.activate = false
+    user
```

Comment:
Does this line actually do anything?

Reply:
I have updated it from `user` to `return user` to make it more explicit.

tl;dr,
- Remove line 58, the function returns `nil`
- Write `user` in line 58, the function returns the local variable user
- Write `return user`, same as above